orchestrator/db/models.py

```diff
@@ -77,25 +77,25 @@ def process_bind_param(self, value: Optional[datetime], dialect: Dialect) -> Opt
         return value
 
     def process_result_value(self, value: Optional[datetime], dialect: Dialect) -> Optional[datetime]:
-        if value is not None:
-            return value.astimezone(timezone.utc)
-        return value
+        return value.astimezone(timezone.utc) if value else value
 
 
 class ProcessTable(BaseModel):
     __tablename__ = "processes"
 
-    pid = Column(UUIDType, server_default=text("uuid_generate_v4()"), primary_key=True, index=True)
-    workflow = Column(String(255), nullable=False)
-    assignee = Column(String(50), server_default=Assignee.SYSTEM, nullable=False)
-    last_status = Column(String(50), nullable=False)
-    last_step = Column(String(255), nullable=True)
-    started_at = Column(UtcTimestamp, server_default=text("current_timestamp()"), nullable=False)
-    last_modified_at = Column(UtcTimestamp, server_default=text("current_timestamp()"), onupdate=nowtz, nullable=False)
-    failed_reason = Column(Text())
-    traceback = Column(Text())
-    created_by = Column(String(255), nullable=True)
-    is_task = Column(Boolean, nullable=False, server_default=text("false"), index=True)
+    pid = mapped_column(UUIDType, server_default=text("uuidmapped_cgenerate_v4()"), primary_key=True, index=True)
```

Comment:
I guess `uuidmapped_cgenerate_v4()` = `uuid_generate_v4()` + accidental paste of `mapped_c`

Reply:
Oops.